narwhals/_arrow/series_list.py

```diff
@@ -6,19 +6,19 @@
 import pyarrow.compute as pc
 
 from narwhals._arrow.utils import ArrowSeriesNamespace
+from narwhals._compliant.any_namespace import ListNamespace
 from narwhals._utils import not_implemented
 
 if TYPE_CHECKING:
     from narwhals._arrow.series import ArrowSeries
 
 
-class ArrowSeriesListNamespace(ArrowSeriesNamespace):
+class ArrowSeriesListNamespace(ArrowSeriesNamespace, ListNamespace["ArrowSeries"]):
     def len(self) -> ArrowSeries:
         return self.with_native(pc.list_value_length(self.native).cast(pa.uint32()))
 
-    unique = not_implemented()
-
-    contains = not_implemented()
-
     def get(self, index: int) -> ArrowSeries:
         return self.with_native(pc.list_element(self.native, index))
+
+    unique = not_implemented()
+    contains = not_implemented()
```

Comment:
Thanks - I was so bothered by this visually but didn't want to open a nitpick PR just for it